README.md

```diff
@@ -65,9 +65,35 @@ register_block_type( 'example-namespace/example-block', array(
 			)
 		)
 	),
+	'render_callback' => 'example_callback',
 ) );
 ```
 
+**Inspector Controls:**
+
+Inspector controls are shown in the sidebar when a block is selected and can easily be added by using `placement: 'inspector'` when adding attributes as shown in the example below.
```

Comment:
Can you include this within the original JS example?

Reply:
@danielbachhuber  Please confirm.

You mean add the inspector control example here in the first JS example?
```js
registerBlockType( 'example-namespace/example-block', {
    title: 'Example Block',
    url: {
        type: 'string',
        field: {
            type: 'url',
        },
    },
    text: {
        type: 'string',
        field: {
            type: 'text',
            placeholder: 'Enter link text',
        },
    },
});
```

Reply:
Yes, add it to the existing example instead of creating a new one. It's more obvious to the end user.

Reply:
Good, although this comment probably no longer applies:

<img width="935" alt="image" src="https://user-images.githubusercontent.com/36432/38199944-583f0bb4-3647-11e8-88dd-367d3f42bf77.png">

Can you remove, or incorporate it as a code comment in the JS example?

Reply:
Added as a code comment.